Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Light/LightBehaviorXaml.bind

```diff
@@ -0,0 +1,32 @@
+<Page
```

Comment:
I think this file is saved using a wrong encoding (the first one in the list is the right one :))